src/pages/events/[eventId].tsx

```diff
@@ -1,12 +1,29 @@
 import { useRouter } from 'next/router';
 import React from 'react';
 
+import { Store } from 'core/redux/Store';
 import DetailView from 'events/components/DetailView';
+import { eventSelectors, fetchEventById } from 'events/slices/events';
+import { NextPageContext } from 'next';
+
+interface IContext extends NextPageContext {
+  store: Store;
+}
 
 const EventDetailPage = () => {
   const router = useRouter();
   const eventId = Number(router.query.eventId);
```

Comment:
Since `getInitialProps` has to return _something_ I returned the `id` in the other examples.
Anything returned from `getInitialProps` will be passed as props to the component, which means we can just use the `eventId` from props instead of getting it from the router again :smile: